infrastructure/lambda/template.yaml

```
@@ -759,6 +765,30 @@ Mappings:
       integration: "https://review-hc.integration.account.gov.uk"
       production: "https://review-hc.account.gov.uk"
 
+  CanaryDeploymentapigatewayidMapping:
+    di-ipv-cri-check-hmrc-api:
+      privateapigw: "check-hmrc-cri-api-PrivateApiGatewayId"
+      publicapigw: "check-hmrc-cri-api-PublicApiGatewayId"
+    di-ipv-cri-address-api:
+      privateapigw: "address-cri-api-v1-PrivateAddressApiGatewayId"
+      publicapigw: "address-cri-api-v1-PublicAddressApiGatewayId"
+    di-ipv-cri-kbv-api:
+      privateapigw: "kbv-cri-private-kbv-cri-api-v1-PrivateKBVApiGatewayId"
+      publicapigw: "kbv-cri-kbv-cri-api-v1-PublicKBVApiGatewayId"
+    # di-ipv-cri-kbv-hmrc-api:
+    #   privateapigw: "kbv-hmrc-cri-api-PrivateApiGatewayId"
+    #   publicapigw: "kbv-hmrc-cri-api-PublicApiGatewayId"
+    # di-ipv-cri-passport-api:
+    #   privateapigw: "ipv-cri-passport-api-PrivateUKPassportAPIGatewayID"
+    #   publicapigw: "ipv-cri-passport-api-PublicUKPassportAPIGatewayID"
+    # di-ipv-cri-dl-api:
+    #   privateapigw: "dl-cri-api-v1-PrivateDlApiGatewayId"
+    #   publicapigw: "dl-cri-api-v1-PublicDlApiGatewayId"
+    # di-ipv-cri-fraud-api:
```

I think we will have to have at least placeholder values for the other CRIs or they will see errors when deploying the stack?

---

enabled the changes for other CRIs